npm/react/package.json

```diff
@@ -37,6 +36,7 @@
     "@babel/preset-typescript": "7.10.4",
     "@bahmutov/cy-api": "1.4.2",
     "@bahmutov/cy-rollup": "2.0.0",
+    "@cypress/webpack-dev-server": "0.0.0-development",
```

Comment:
Will this work for the published package? Does semantic release automatically inject correct version?

Reply:
Since webpack-dev-server is not published yet, this is the only version that yarn will accept. 
I believe the dependencies to be updated by server automatically at release but I will check the script.

It's better to be safe than sorry.

Reply:
I just checked, and it does do it for webpack-preprocessor. 
We should be OK. 
We can always deliver a new fixed version later by merging on master if it was not the case.